fontforge/splinestroke.c

```diff
@@ -1303,7 +1303,7 @@ static bigreal SplineStrokeNextT(StrokeContext *c, Spline *s, bigreal cur_t,
 
     next_ut = SplineStrokeNextAngle(c, *cur_ut, is_ccw, &next_curved,
                                     reverse, nci_hint);
-    next_t = SplineSolveForUTanVec(s, next_ut, cur_t);
+    next_t = SplineSolveForUTanVec(s, next_ut, cur_t, false);
```

Comment:
Forgive my ignorance, but it seems `picky` is never `true` - why PR this? You call it &laquo;optional&raquo;, but I don't see a way to enable it without editing this line (and the previous similar line)?

Reply:
I wrote `SplineSolveForUTanVec()` as a "general" facility that could be called in the future by other projects. I was nervous about adding this workaround for general use so I added a boolean to control whether it applies. 

When `picky` is passed as true the function works as it did before (`if (picky) return -1.0;`). As it happens I don't need that in the two places I call it -- I want the looser fallback if it's present.

Reply:
So, it's optional in the sense that future consumers of the function can avoid it.

Reply:
OK, I understand. In that case I'll go ahead and merge this, you know best what's needed, and it's essentially a no-op in the current code.

Reply:
Well, as it happens the only two times this function is called have now changed, in that there is a section of `SplineSolveForUTanVec()` that looks for looser (i.e. less accurate) slope matches. 

But only after the tighter matches have failed, and the looser ones still have to be pretty close.